### PR TITLE
Sandbox transfert

### LINKED ISSUE #68: Add the script to scrap lipid database (CSML and MAD) in a repertory `script` and update the README

### PR DESCRIPTION
**Pull request by KarinDuong**
close #68

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,3 +54,18 @@ or with the URL:
...
+```bash
+python script/scrap_MAD.py
+```
+
+And run the script to download all-atom model molecule data: 
+```bash
+python script/scrap_charmm_gui_CSML.py
+```
+
+These scripts analyzes the database MAD (https://mad.ibcp.fr/explore) and CHARMM-GUI CSML (https://www.charmm-gui.org/?doc=archive&lib=lipid). It download the data - if it's not already exist. It extracts information that can be used to identify lipids in the GRO and PDB files and save it into a CSV file.
```
Comment (pierrepo) [inline README.md:70]: Could you please create subsections for both databases? And also mention explicitly the name of database file.

**Comment by KarinDuong** [comment]
Like that ? 
# Update database 
## CSML 
[Information/description]

## MAD
[information/description]

File: README.md (hunk 1 above)
Comment (KarinDuong) [reply README.md:70]: Like that ? 
# Update database 
## CSML 
[Information/description]

## MAD
[information/description]

Comment (pierrepo) [reply README.md]: yes

**Comment by pierrepo** [comment]
Could you put these Python scripts in the `scripts` folder (instead of `script`)?

**Comment by KarinDuong** [comment]
Done :white_check_mark:

**Pull request merged**